api/app/routers/hfi_calc.py

```diff
@@ -145,19 +168,43 @@ def extract_selected_stations(request: HFIResultRequest) -> List[int]:
 
 
 @router.post("/fire_centre/{fire_centre_id}/{start_date}/planning_area/{planning_area_id}"
-             "/station/{station_code}/enable/{enable}")
+             "/station/{station_code}/selected/{enable}")
```

Comment:
Should the route path be `toggle` instead of `selected` to match the function it decorates, or vice versa?

Reply:
It's not toggle. That was my first though, but toggle only happens right at the top in the ui - from there down it's select true or select false.

Reply:
Ok please just rename the function so it's consistent with the route

Reply:
Done